R/youth_allowance.R

```diff
@@ -0,0 +1,198 @@
+#' @param is_single Is the individual single?
+#' @param living_at_home Does the individual live at home with their parents?
+#' @param has_children Does the individual have any children?
+#' @param isjspceoalfofcoahodeoc Is the recipient a single job seeker principal carer, either of large family or foster child/ren, or who is a home or distance educator of child/ren?
+#' @param is_student Is the individual a student? Note that apprenctices are considered students.
+#' @param per How often the payment will be made. Default is fortnightly. At present payments can only be fortnightly.
+#' @param taper1 The amount at which the payment is reduced for each dollar earned between the lower and upper bounds.
+#' @param taper2 The amount at which the payment is reduced for each dollar earned above the upper bound.
+#' @param FT_YA_student_lower Student and apprentice lower bound for which reduction in payment occurs at rate taper1
+#' @param FT_YA_student_upper Student and apprentice upper bound for which reduction in payment occurs at rate taper1. Student and apprentice lower bound for which reduction in payment occurs at rate taper2.
+#' @param FT_YA_jobseeker_lower Jobseeker lower bound for which reduction in payment occurs at rate taper1
+#' @param FT_YA_jobseeker_upper Jobseeker upper bound for which reduction in payment occurs at rate taper1. Student and apprentice lower bound for which reduction in payment occurs at rate taper2.
+#' @param excess_partner_income_mu The amount at which the payment is reduced for each dollar earned by the individual's partner.
+#' @export youth_allowance 
+
+
+
+youth_allowance <- function(ordinary_income = 0,
+                            fy.year = yr2fy(year(Sys.Date())),
+                            age = 18,
+                            eligible_if_over22 = FALSE,
+                            is_single = TRUE,
+                            living_at_home = FALSE,
+                            has_children = FALSE,
+                            isjspceoalfofcoahodeoc = FALSE,
+                            is_student = TRUE,
+                            per = "fortnight",
+                            taper1 = 0.5,
+                            taper2 = 0.6,
+                            FT_YA_student_lower = 437,
+                            FT_YA_student_upper = 524,
+                            FT_YA_jobseeker_lower = 143,
+                            FT_YA_jobseeker_upper = 250,
+                            excess_partner_income_mu = 0.6) {
+  stopifnot(per == "fortnight")
+  
+  #max rate data: http://guides.dss.gov.au/guide-social-security-law/5/1/1/20
+  #note: identical for student and jobseeker
+  max_rate_March_2018 <-
+    if_else(ordinary_income >= Inf,
+            NA_real_,
```

Comment:
Why?

Reply:
Honestly not too sure, that was there from when you did it. Happy to remove it.